apps/humanatlas.io/src/app/pages/landing-page/landing-page.component.ts

```diff
@@ -0,0 +1,20 @@
+import { ChangeDetectionStrategy, Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { HraCommonModule } from '@hra-ui/common';
+import { PageSectionComponent } from '@hra-ui/design-system/content-templates/page-section';
+
+import { CarouselComponent } from '../../components-v2/carousel/carousel.component';
+import { CountInfoComponent } from '../../components-v2/count-info/count-info.component';
+import { SectionCardsComponent } from '../../components-v2/section-cards/section-cards.component';
+
+/**
+ * HRA landing page component
+ */
+@Component({
+  selector: 'hra-landing-page',
+  imports: [HraCommonModule, CarouselComponent, CountInfoComponent, SectionCardsComponent, PageSectionComponent],
+  templateUrl: './landing-page.component.html',
+  styleUrl: './landing-page.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
You should be able to remove this

Reply:
Please remove

Reply:
The page section component doesn't work without this, is it ok to leave it in for now?